pipelines/team-tasks.yml

```
@@ -0,0 +1,143 @@
+members: &members
+# Aidan
+- "<@W013S2H8ABU>"
+# Scott
+- "<@W013B9SAXJT>"
+# Daniel
+- "<@W013QLP3YJZ>"
+# Esteban
+- "<@U01LJULP0LS>"
+# Alex
+- "<@W013S7H1JCT>"
+# Rui
+- "<@W012PK4PCR1>"
+# Taylor
+- "<@W011PN57Q8Y>"
+# Muntasir
+- "<@W011EC91F5L>"
+# Navdeep
+- "<@W013K986P7G>"
+# Matthew
+- "<@W013BA1CNMD>"
+# Clara
+- "<@W013B99CVU7>"
+
+resource_types:
+- name: slack-notification
+  type: registry-image
+  source:
+    repository: cfcommunity/slack-notification-resource
+
+- name: static-list-resource
+  type: registry-image
+  source:
+    repository: taylorsilva/static-list-resource
+
+- name: cron-resource
+  type: registry-image
+  source:
+    repository: cftoolsmiths/cron-resource
+
+resources:
+- name: slack-message
+  type: slack-notification
+  source:
+    url: ((slack_hooks.concourse-private))
+
+- name: first-day-of-month
+  type: cron-resource
+  source:
+    expression: "0 0 1 * *"
+    location: "America/Toronto"
+    fire_immediately: true
+
+- name: good-morning
+  type: time
+  source:
+    start: 8:00 AM
+    stop: 9:00 AM
+    location: "America/Toronto"
+    days: [Monday, Tuesday, Wednesday, Thursday, Friday]
+
+- name: friday-morning
+  type: time
+  source:
+    start: 11:00 AM
+    stop: 11:30 AM
+    location: "America/Toronto"
+    days: [Friday]
+
+- name: lunch
+  type: static-list-resource
+  source:
+    unique: lunch
+    list: *members
+
+- name: standup
+  type: static-list-resource
+  source:
+    unique: standup
+    list: *members
+
+- name: retro
+  type: static-list-resource
+  source:
+    unique: retro
+    list: *members
+
+jobs:
+- name: monthly-team-lunch
+  plan:
+  - get: first-day-of-month
+    trigger: true
+  - get: lunch
+  - put: slack-message
+    params:
+      text_file: lunch/item
+      text: |
+        $TEXT_FILE_CONTENT will organize this month's team lunch!
+        1. Set a date/time and create a calendar event
+        2. Think of something the team can do/watch (e.g. codenames)
+  - put: next-months-organizer
+    resourc: lunch
```

Just a small spelling mistake!

```suggestion
    resource: lunch
```